send-email-advanced/send-email-continuation-token/send-email-continuation-token.py

```diff
@@ -0,0 +1,47 @@
+from azure.communication.email import EmailClient
+
+connection_string = "<ACS_CONNECTION_STRING>"
+sender_address = "<SENDER_EMAIL_ADDRESS>"
+recipient_address = "<RECIPIENT_EMAIL_ADDRESS>"
+
+POLLER_WAIT_TIME = 10
+
+message = {
+    "senderAddress": sender_address,
+    "recipients":  {
+        "to": [{"address": recipient_address}],
+    },
+    "content": {
+        "subject": "Test email from Python Sample",
+        "plainText": "This is plaintext body of test email.",
+        "html": "<html><h1>This is the html body of test email.</h1></html>",
+    }
+}
+
+try:
+    client = EmailClient.from_connection_string(connection_string)
+    poller = client.begin_send(message);
+
+    # Pauses operation and saves state that can be used later to resume operation
+    token = poller.continuation_token()
+
+    new_client = EmailClient.from_connection_string(connection_string);
+    new_poller = new_client.begin_send(message, continuation_token=token);
+
+    time_elapsed = 0
+    while not new_poller.done():
+        print("Email send poller status: " + new_poller.status())
+
+        new_poller.wait(POLLER_WAIT_TIME)
+        time_elapsed += POLLER_WAIT_TIME
+
+        if time_elapsed > 18 * POLLER_WAIT_TIME:
```

Comment:
What does 18 represent here? And why specifically 18? Might be good to have a constant that describes what this number represents

Reply:
It's just an arbitrary number for the max number of times the poller should poll (180 seconds / 3 minutes). I'll make this a constant.